lib/web_ui/test/engine/surface/scene_builder_test.dart

```diff
@@ -525,6 +525,38 @@ void testMain() {
     await testCase('be', 'remove in the middle', deletions: 2);
     await testCase('', 'remove all', deletions: 2);
   });
+
+  test('Canvas should allocate fewer pixels if scaled', () async {
```

Comment:
scaled as in "zoomed in" or "zoomed out"? (I know the answer now, but will need to think longer in a few months)

Reply:
Updated comment. thx.